R/fda-table_04.R

```diff
@@ -30,32 +28,35 @@
 #'   ) %>%
 #'   mutate(DCTREAS = DCSREAS)
 #'
-#' tbl <- make_table_04(df = adsl)
+#' tbl <- make_table_04(
+#'   df = adsl, pop_var = c("RANDFL", "ITTFL", "SAFFL", "PPROTFL"),
+#'   pop_var_lbl = c("Patients randomized", "ITT/mITT population", "Safety population", "Per-protocol population")
+#' )
 #' tbl
 #'
 #' @export
 make_table_04 <- function(df,
                           alt_counts_df = NULL,
                           show_colcounts = TRUE,
                           arm_var = "ARM",
-                          saffl_var = "SAFFL",
+                          pop_var = c("SAFFL"),
+                          pop_var_lbl = c("Safety population"),
```

Comment:
To align more closely with other label variables in the package
```suggestion
                          lbl_pop_vars = c("Safety population"),
```